modules/openapi-generator/src/main/resources/C-libcurl/apiClient.c.mustache

```diff
@@ -134,6 +134,15 @@ void apiClient_free(apiClient_t *apiClient) {
     free(apiClient);
 }
 
+void apiClient_free_data(apiClient_t *apiClient) {
+    if (apiClient->dataReceived) {
```

Comment:
I don't see where this function is being called.  What's its purpse ?  for free-form object parsing ?